interim-jan-2021/resolver-behavior-self-description.md

```diff
@@ -0,0 +1,27 @@
+# Resolver behavior self-description
+
+## Defining local-only namespaces
+* Express namespaces which only this resolver can resolve
+* Ex 1: Enterprise resolvers serving corporation-specific namespaces
+* Ex 2: Public Wi-Fi or cellular networks offering network-local services
+
+## Defining per-namespace optimization
+* Express namespaces for which this resolver provides preferable resolutions
+* Ex 1: ISP routes public content requests to network caches
```

Comment:
Is this really defined by a namespace?  I don't suppose you're going to enumerate all of Cloudflare's domains...

Reply:
Downstream CDN is deployed in the ISP cloud for better user experience. For example, 5G discusses deploying downstream CDN (e.g., video streaming content, gaming servers) in the edge. 

I don't think it is feasible to provide all the domains for which downstream CDN is deployed in the ISP cloud (edge or core).

Reply:
I mean, feasible or not, a resolver ought to be able to inform clients that it has unique enlightenment for some namespaces for routing to local caches.